simple_config

```diff
@@ -1,5 +1,5 @@
 # This is a comment
-port 80;
+port 2020;
```

Comment:
just in case, please change this back to port 80 since our deployment script exposes port 80 on the vm